### DISCUSSION TITLE
is jimmy's code still private??

### DISCUSSION THREAD

**Discussion by mnnkhndlwl** [ankushgarg1998/ipu-ranklist#151] [Q&A]
can i get the access to that code

**Comment by mnnkhndlwl** [comment]
i had build a python script that worked on the old pdfs but now ip has changed it's pdf's ui now i want to know how you extract results from every kind of pdf

**Comment by ankushgarg1998** [comment]
> Is Jimmy's code still private?

Yes.

> Can I get access to that code?

No. 
Although I might make an exception if I knew the use-case you intend to use it for.

> How do you extract results from every kind of PDF?

Jimmy has brothers. :D

**Reply by mnnkhndlwl** [reply]
So there's a different code for different PDFs??

**Comment by mnnkhndlwl** [comment]
I am making a student's application for my college as my project for major or minor so that students can see their attendance , notices , datesheets , class timetables and other more features so I want that app to show student's their marks in that app that's why I was asking
